theano/tensor/opt.py

```diff
@@ -1677,6 +1677,7 @@ def local_fill_sink(node):
 register_canonicalize(local_fill_sink)
 
 
+@register_canonicalize("fast_compile")
```

Comment:
Ah! I'm sorry about this. The email from github showed the function before this. I'll add it at the correct place